tools/ovc/openvino/tools/ovc/cli_parser.py

```diff
@@ -646,7 +646,7 @@ def get_common_cli_parser(parser: argparse.ArgumentParser = None):
     # Command line tool specific params
     common_group.add_argument('--output_model',
                               help='This parameter is used to name output .xml/.bin files with converted model.')
-    common_group.add_argument('--compress_to_fp16', action='store_true',
+    common_group.add_argument('--compress_to_fp16', type=check_bool, default=True,
```

Comment:
@popovaan, @pavel-esir, how is it going to be printed in the help for ovc tool?

Reply:
```
  -h, --help            show this help message and exit

Framework-agnostic parameters:
  --output_model OUTPUT_MODEL
                        This parameter is used to name output .xml/.bin files with converted model.
  --compress_to_fp16    Compress weights in output IR .xml/bin files to FP16.
```

probably we need it to be informative, state explicitly that user can specify True/False flags and that by default it is True

Reply:
--compress_to_fp16 COMPRESS_TO_FP16
                        Compress weights in output IR .xml/bin files to FP16.

Reply:
Yes, please check this PR:
https://github.com/openvinotoolkit/openvino/pull/18785